### PR TITLE
AMP liveblog styling improvements

### PR DESCRIPTION
**Pull request by nicl**
## What does this change?

Quite a few styling fixes:

* styling for pagination (the initial aim and main bit of this)
* misc other styling fixes for text element stuff (headings, blockquote tweaks, list styling fixes, ...)

## Why?

https://trello.com/c/NWdGnZsy

To get to 100%!

Screenshot 2019-06-04 at 16 56 01
Screenshot 2019-06-04 at 17 36 56
Screenshot 2019-06-04 at 17 40 53
Screenshot 2019-06-04 at 17 40 58

### CODE REVIEW

File: packages/frontend/amp/components/Pagination.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    position: relative;
+    color: ${palette.neutral[7]};
+    border: 1px solid ${palette.neutral[86]};
+    height: 36px;
+    line-height: 38px;
+    display: inline-block;
+
+    margin-right: ${isMarginRight ? '5px' : '0px'};
+
+    span {
```
Comment (gtrufitt) [inline packages/frontend/amp/components/Pagination.tsx]: Can we create a named variable for this one just for clarity?

File: packages/frontend/amp/components/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            hasRightMargin,
+        );
+
+        const attrs = {
+            className: styles,
+            href: suffix ? url + suffix : undefined,
+        };
+
+        return (
+            <a {...attrs}>
```
Comment (gtrufitt) [inline packages/frontend/amp/components/Pagination.tsx]: I assume `href` doesn't exist if `undefined` - how does that handle keyboard tabbing (bearing in mind that mobile screen readers would use similar tab index for navigation)?

File: packages/frontend/amp/components/Pagination.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                {link(
+                    guardianURL,
+                    <ChevronLeftDouble />,
+                    pagination.newest,
+                    true,
+                )}
+                {link(guardianURL, <ChevronLeftSingle />, pagination.newer)}
+            </span>
+
+            <span>
```
Comment (gtrufitt) [inline packages/frontend/amp/components/Pagination.tsx]: I wonder if this should be a `p` semantically (and from a focusable point of view)?

File: packages/frontend/amp/components/elements/Text.tsx [hunk 4]
```diff
@@ -31,7 +32,32 @@ export const TextStyle = (pillar: Pillar) => css`
...
+
+    li {
+        margin-bottom: 6px;
+        padding-left: 20px;
+        p {
+            display: inline;
+        }
+    }
+
+    li:before {
```
Comment (gtrufitt) [inline packages/frontend/amp/components/elements/Text.tsx]: Just for quick reference. (HAVE I USED THE RIGHT COMMENT 😨)

```suggestion
	/* Bullet point */
    li:before {
```

File: packages/frontend/amp/components/Pagination.tsx (hunk 2 above)
Comment (nicl) [reply packages/frontend/amp/components/Pagination.tsx]: Ah interesting, I am not sure how it handles tabbing. I mirrored the markup on prod/mobile I think but will have a think and check the attributes there.

Comment (nicl) [reply packages/frontend/amp/components/Pagination.tsx]: okay, so https://stackoverflow.com/questions/10510191/valid-to-use-a-anchor-tag-without-href-attribute/10510353 seems to suggest it is okay:

* it's valid html
* it doesn't fall into the tab order if no href present

Comment (gtrufitt) [reply packages/frontend/amp/components/Pagination.tsx]: Thanks for checking!

**Pull request merged**